### PR TITLE
Issue 619: Convert contributing and roadmap docs to asciidoc

### PR DESCRIPTION
**Pull request by thatdocslady**
Fixes issue #619 

Converted the files to asciidoc and adjusted some light proofreading and formatting.

Out of scope: Linking to these files from elsewhere (only one reference was found and it's in the README which will already be handled as a part of #533 ).

### CODE REVIEW

File: ROADMAP.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+The following sections give a high level overview of the envisioned
+features and how they fit into upcoming releases. Of course this is
+subject to change and actual version names might change as well, because
+http://semver.org/[semantic versioning] is used to determine the final
+version name of a given release.
+
+[[minishift-100]]
+=== Minishift 1.0.0
+
+- Ability to select the OpenShift version of the cluster [line-through]#link:https://github.com/minishift/minishift/issues/316[#141]#
```
Comment (hferentschik) [inline ROADMAP.adoc]: You lost some formatting here. In Markdown completed features used strikethrough. Now there is no indication anymore what is completed. If not strikethrough, we might now use checkbox '- [ ] ' syntax which is quite nice as well. All expect "Consistent host folder mounting https://github.com/minishift/minishift/issues/316[#316], https://github.com/minishift/minishift/issues/317[#317]" should be marked completed.

File: ROADMAP.adoc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- Consistent host folder mounting https://github.com/minishift/minishift/issues/316[#316], https://github.com/minishift/minishift/issues/317[#317]
+- Exposure of the OpenShift registry https://github.com/minishift/minishift/issues/254[#254]
+- Ability to configure/patch OpenShift's master and node configuration [line-through]#https://github.com/minishift/minishift/issues/276[#276]#
+- Ability to customize cluster (for example, adding templates and imagestreams) https://github.com/minishift/minishift/issues/177[#177]
+
+[[minishift-110]]
+=== Minishift 1.1.0
+
+- Add 'minishift login' command https://github.com/minishift/minishift/issues/349[#349]
+- Add 'minishift' context https://github.com/minishift/minishift/issues/359[#359]
```
Comment (hferentschik) [inline ROADMAP.adoc]: This should be moved to the 1.0 section and marked as completed.

File: ROADMAP.adoc (hunk 1 above)
Comment (thatdocslady) [inline ROADMAP.adoc]: Oh damn. Well the [line-through] tag is the asciidoc way of marking strikethrough, and I tested this locally fine, but now after submitting the PR I see that it doesn't live-render in GitHub. Need to think about how to mitigate, maybe we can just write something like [DONE] next to features that were completed instead?

**Comment by thatdocslady** [comment]
@hferentschik the checkbox formatting works so I'll go ahead and apply it instead of the strikethrough

**Comment by hferentschik** [comment]
> the checkbox formatting works so I'll go ahead and apply it instead of the strikethrough

works for me

**Comment by hferentschik** [comment]
Merged, thanks

**Pull request closed without merging**